### PR TITLE
Governance: Change timestamps and time periods to UnixTimestamp

### PR DESCRIPTION
**Pull request by SebastianBor**
#### Summary 
Change all `Proposal` lifetime timestamps and configurable time periods (`voting_time`, `instruction_hold_up_time`) from slots to UnixTimestamp to make them human friendly

### CODE REVIEW

File: governance/program/tests/program_test/mod.rs [hunk 1]
```diff
@@ -1505,6 +1508,39 @@ impl GovernanceProgramTest {
...
+            .await
+    }
+
+    #[allow(dead_code)]
+    pub async fn advance_clock_past_timestamp(&mut self, unix_timestamp: UnixTimestamp) {
+        let mut clock = self.get_clock().await;
+        let mut n = 1;
+
+        while clock.unix_timestamp <= unix_timestamp {
+            // Since the exact time is not deterministic keep wrapping by arbitrary 400 slots until we pass the requested timestamp
```
Comment (joncinque) [inline governance/program/tests/program_test/mod.rs:1523]: Does this actually work to advance the timestamp the amount expected?  When we warp, the new bank is supposed to calculate the new timestamp based on voting activity, but without voting activity, I'm not sure how it calculates.

**Review by joncinque — APPROVED** [review]
Looks good, just the one question about advancing the clock, since I didn't think that was possible!

File: governance/program/tests/program_test/mod.rs (hunk 1 above)
Comment (SebastianBor) [reply governance/program/tests/program_test/mod.rs:1523]: Yes, it does work. I've looked into the code and I don't fully understand it but the updated timestamp is calculated by [get_timestamp_estimate](https://github.com/solana-labs/solana/blob/c1b9d40a64e57375e4298d8d96345a4bb17b5de3/runtime/src/bank.rs#L2083) and I empirically confirmed that for the given starting conditions (slot, epoch, warp_to_slot) the result is deterministic. 
vote_accounts() is not empty in that call however the timestamp is moved by the drift correction logic.

Comment (joncinque) [reply governance/program/tests/program_test/mod.rs:1523]: Ah, very cool -- nice job!

**Pull request merged**